### ISSUE TITLE
ARIA jQuery Carousel: Keyboard nav conflict and several  “list blank”

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1759]
**Reported by kevinchao89 on 2011-09-07 02:37**
Firefox Nightly 9.0a1 (2011-09-06) and NVDA Main Snapshot 4634

Firefox: http://hanshillen.github.com/jqtest/#goto_carousel
Focused on graphic, should be able to use LEFT/RIGHT ARROW to go to next/previous item, but NVDA goes to next/previous character;
NVDA+F2 for pass-through, RIHGT ARROW will go to next item, switch to focus mode, where LEFT/RIGHT will work;
ESC to switch to browse mode, UP/DOWN ARROW will result in NVDA reading: list blank, where it should read different items.

**Comment by bhavyashah** [comment]
I was unable to reproduce all issues reported with NVDA 2017.2 and Firefox 54 stable.
* Arrowing around the first graphic on the provided URL did indeed navigate only between the characters of the alt text or description of the image. However, I am unsure of whether this should be considered expected behaviour, or if the graphic should occupy only one character in terms of arrowing and its corresponding text be accessible via object review.
* Contrary to the the ticket's report, NVDA+F2 also navigated within the image text itself rather than moving to the next item.
* I was slightly confused as to what to expect and what happened when testing this in focus mode.
* Arrowing up and down back in browse mode reported as expected as opposed to Kevin's report of 'list blank'.
Since these sound like multiple issues which seem to have been fixed either in Firefox or in NVDA overtime, we do not have any means to contact the original reporter, and this issue is almost six years old now, I suggest closing. @ehollig

**Comment by jcsteh** [comment]
I think the issue as reported here is invalid/works for me. The graphic isn't focusable, and even if it were, a graphic isn't an interactive widget according to spec. The left and right arrows do work in the list in focus mode, and if you tab to the list, NVDA switches to focus mode automatically as it should.
